app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -270,6 +270,34 @@ function handleKeyUp(event) {
   handleSizeChange(event);
 }
 
+function handleKeyDown(event) {
+    // If Ctrl+Enter OR Command+Enter is pressed
+    if((event.ctrlKey || event.metaKey) && event.keyCode === 13)
+    {
+        // Get user details and extract code of conduct & comment count
+        var user = userData();
+        if (!user){
+            return;
+        }
+        //var codeOfConduct = user.checked_code_of_conduct;
```

Comment:
Why did you commented out this? Shouldn't we use it? In that case could we remove it?